sql/derived_tables/invoice_lines_fund_distributions.sql

```diff
@@ -0,0 +1,66 @@
+
+DROP TABLE IF EXISTS local.invoice_lines_fund_distributions;
+
+-- Create a local table for Fund Distributions
+
+CREATE TABLE local.invoice_lines_fund_distributions AS
+WITH funds_distr AS (
+SELECT
+    id AS invoice_line_id,
+	json_extract_path_text(json_array_elements(json_extract_path(data, 'fundDistributions')), 'code') AS fund_distribution_code,
+    json_extract_path_text(json_array_elements(json_extract_path(data, 'fundDistributions')), 'fundId') AS fund_distribution_id,
+	json_extract_path_text(json_array_elements(json_extract_path(data, 'fundDistributions')), 'distributionType') AS fund_distribution_type,
+	json_extract_path_text(json_array_elements(json_extract_path(data, 'fundDistributions')), 'value') AS fund_distribution_value,
```

Comment:
For next time please use the new cross join method shown in https://github.com/library-data-platform/ldp/blob/master/doc/User_Guide.md#7-json-arrays .  I will fix this one.  Thank you.